packages/@lwc/compiler/package.json

```diff
@@ -18,6 +18,7 @@
         "@babel/plugin-proposal-object-rest-spread": "7.0.0",
         "@lwc/babel-plugin-component": "1.0.2",
         "@lwc/errors": "1.0.2",
+        "@lwc/shared": "1.0.2",
```

Comment:
@ekashida you fixed some of this few days ago... you explicitly made shared private, but this PR is making it public, which is not ideal since we don't want people to rely on that pkg directly, only us.

Reply:
Nothing works in several packages because this is used in the main code...